src/gql/queries/project.ts

```diff
@@ -50,6 +70,13 @@ export const GET_PROJECTS = gql`
           id
         }
       }
```

Comment:
Same as already mentioned, if this nodes { ... } is not needed anymore, remove it.